example/src/styles/index.scss

```diff
@@ -0,0 +1,8 @@
+$text-color: #444;
+$background: #eee;
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  color: $text-color;
+  background-color: $background;
+}
```

Comment:
@byhag 
![image](https://user-images.githubusercontent.com/12001504/70064634-797e4580-15af-11ea-94dd-1bd79d7e9bb0.png)

This is probably a problem with `@jbknowledge/eslint-config`. can you investigate and create a new issue on that repo if needed?

Reply:
fixed